src/collection/pages/main/main.tsx

```diff
@@ -161,11 +161,14 @@ const Main: React.FC = () => {
     }
   }, [collectionState, data])
 
-  if (loading) return <LoadingPage/>;
+  if (loading || minerAddressLoading) return <LoadingPage/>;
   if (error) return <p>error</p>;
   if(data?.stateQuery.agent == null) {
     <div>you need create avatar first</div>
   }
+  if(minerAddress?.minerAddress == null) {
+    <div>you need login first</div>
```

Comment:
1. Is it okay that we don't have `return`?
2. it would be better if we have `FIXME` like as...

```
```
```suggestion
    // FIXME we should translate this message.
    <div>you need login first</div>
```